Dockerfile

```diff
@@ -4,9 +4,9 @@ WORKDIR /shotover-proxy
 
 COPY ./ ./
 
-RUN cargo build --release
+RUN cargo build -p shotover-proxy --release
 
-FROM debian:bullseye-slim
+FROM debian:stable-slim
```

Comment:
The rust and debian images need to be pinned to the same debian release or we will hit the libc issue again.

Reply:
Alright, I'll pin both to bookworm - latest stable